src/server/plugins/engine/types.ts

```diff
@@ -71,19 +71,18 @@ export interface FormSubmissionError
   text: string // e.g: 'Date field must be a real date'
 }
 
+export interface FormParams {
+  action?: FormAction
+  confirm?: true
```

Comment:
@colinrotherham should this be `boolean`?

Reply:
Glad you spotted this

Remember the `confirm === true` PR feedback the other day?

You can't submit `confirm=false` so it's only ever `true` or `undefined` in the HTML

Reply:
aha 💡